src/spatial_ref/srs.rs

```diff
@@ -506,6 +506,23 @@ impl SpatialRef {
         res
     }
 
+    #[cfg(any(major_ge_4, all(major_ge_3, minor_ge_1)))]
+    pub fn to_projjson(&self) -> Result<String> {
+        let mut c_projjsonstr = ptr::null_mut();
+        let options = ptr::null();
```

Comment:
In the long run, I'd like us to support options where applicable:

https://gdal.org/doxygen/classOGRSpatialReference.html#a1a4b65b551d3a5ea91fb7fd8d9e1e4e8